Extensions/dnSpy.Debugger/dnSpy.Debugger.DotNet.Metadata/DmdLazyMetadataBytes.cs

```diff
@@ -151,4 +151,15 @@ public sealed class DmdLazyMetadataBytesCom : DmdLazyMetadataBytes {
 			Dispatcher = dispatcher ?? throw new ArgumentNullException(nameof(dispatcher));
 		}
 	}
+	/// <summary>
```

Comment:
```suggestion

	/// <summary>
```
Reason: New line between class definitions, consistency.